bamboo/set-bamboo-env-variables.sh

```diff
@@ -114,6 +114,7 @@ if [[ $bamboo_NGAP_ENV = "SIT" ]]; then
   export VPC_CIDR_IP=$bamboo_SECRET_SIT_VPC_CIDR_IP
   export PROVIDER_HOST=$bamboo_SECRET_SIT_PROVIDER_HOST
   export SECURITY_GROUP=$bamboo_SECRET_SIT_SECURITY_GROUP
+  export SHARED_LOG_DESTINATION_ARN=$bamboo_SIT_SHARED_LOG_DESTINATION_ARN
```

Comment:
Unrelated?

Reply:
related to fixing SIT yes, because we have to have different endpoints for the shared log destination based on the metrics account ID.